extensions/wikia/Flags/FlagsExtractor.class.php

```diff
@@ -531,6 +524,39 @@ public function findTemplatePosition( $templateName, $offset ) {
 		return false;
 	}
 
+	private function getNextTemplateFormat() {
```

Comment:
I found this name confusing, since it's about a current template. What about simply `getTemplateFormat`? :)